README.md

```diff
@@ -58,6 +60,12 @@ Environment variables:
 * `TRELLO_TEST_PASSWORD`
     * required
     * Password to use while running cukes
+* `INTEGRATION_KEY`
+    * required
+    * same as your `PUBLIC_KEY` up above, but is used for integration specs
+* `INTEGRATION_TOKEN`
```

Comment:
The problem with that is the integration tests write to the board as well (so the test isn't brittle).